cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -1,37 +1,28 @@
-const $ = require( 'jquery' );
-const amortize = require( 'amortize' );
+
+import $ from 'jquery'
+import './tab';
+import 'rangeslider.js';
+import * as params from './params';
+import * as template from './template-loader';
+import amortize from 'amortize';
+import config from '../../config.json';
+import dropdown from '../dropdown-utils';
+import fetchRates from '../rates';
 import formatUSD from 'format-usd';
-const isNum = require( 'is-money-usd' );
-const jumbo = require( 'jumbo-mortgage' );
-const median = require( 'median' );
+import Highcharts from 'highcharts';
+import HighchartsExport from 'highcharts/modules/exporting';
+import isNum from 'is-money-usd';
+import jumbo from 'jumbo-mortgage';
+import median from 'median';
 import unFormatUSD from 'unformat-usd';
-import {
```

Comment:
This should be retained.

Reply:
yeah, I'm fixing this. I screwed up the rebase.